hikari/channels.py

```diff
@@ -531,6 +531,7 @@ async def send(
         embeds: undefined.UndefinedOr[typing.Sequence[embeds_.Embed]] = undefined.UNDEFINED,
         tts: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
         reply: undefined.UndefinedOr[snowflakes.SnowflakeishOr[messages.PartialMessage]] = undefined.UNDEFINED,
+        fail_if_not_exists: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
```

Comment:
Potential idea, but would `reply_fail_if_not_exists` be a better name so that its not confusing?

Reply:
this might be okay 🤔

Reply:
Lets go for it then!